Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/Resources/RazorTargetTestsCaptureDataTargetsFileTemplate.xml

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version='1.0' encoding='utf-8'?>
+<Project ToolsVersion='Current' xmlns='http://schemas.microsoft.com/developer/msbuild/2003'>
+
+  <Target Name='CaptureValues' AfterTargets='{0}'>
+    <Message Importance='high' Text='CAPTURE::PROPERTY::TargetDir::$(TargetDir)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::TargetFileName::$(TargetFileName)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::ErrorLog::$(ErrorLog)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::ProjectConfFilePath::$(ProjectConfFilePath)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::ResolvedCodeAnalysisRuleSet::$(ResolvedCodeAnalysisRuleSet)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::TreatWarningsAsErrors::$(TreatWarningsAsErrors)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::WarningsAsErrors::$(WarningsAsErrors)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::WarningLevel::$(WarningLevel)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::ProjectSpecificOutDir::$(ProjectSpecificOutDir)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::ProjectSpecificConfDir::$(ProjectSpecificConfDir)' />
+
+    <!-- Item group values will be written out one per line -->
+    <Message Importance='high' Text='CAPTURE::ITEM::AdditionalFiles::%(AdditionalFiles.Identity)' Condition=" @(AdditionalFiles) != '' "/>
+    <Message Importance='high' Text='CAPTURE::ITEM::Analyzer::%(Analyzer.Identity)'  Condition=" @(Analyzer) != '' " />
+
+    <Message Importance='high' Text='CAPTURE::PROPERTY::RazorSonarCompileErrorLog::$(RazorSonarCompileErrorLog)' />
+    <Message Importance='high' Text='CAPTURE::PROPERTY::RazorCompilationErrorLog::$(RazorCompilationErrorLog)' />
```

Comment:
Does this need it's separate XML file? Can't you just add this to the `RoslynTargetTestsCaptureDataTargetsFileTemp`?

Or, does this file really need all present captures, as the test is interested only in 3 specific properties?

I don't know if there are some unwanted consequences. I guess it will capture more stuff that other tests will not care about.